DIPs/8NNN-WGB.md

```diff
@@ -0,0 +1,68 @@
+# Deprecate Brace-Style Struct Initializers
+
+| Field           | Value                                                           |
+|-----------------|-----------------------------------------------------------------|
+| DIP:            | (number/id -- assigned by DIP Manager)                          |
+| Review Count:   | 0 (edited by DIP Manager)                                       |
+| Author:         | Walter Bright walter@digitalmars.com                            |
+| Implementation: | (links to implementation PR if any)                             |
+| Status:         | Will be set by the DIP manager (e.g. "Approved" or "Rejected")  |
+
+## Abstract
+
+With the adoption of named argument lists, the brace-style struct initializer
+syntax is redundant with that of function-style struct literals. Removing
+the brace-style will reduce the complexity of the language.
+
+## Contents
+* [Rationale](#rationale)
+* [Prior Work](#prior-work)
+* [Description](#description)
+* [Breaking Changes and Deprecations](#breaking-changes-and-deprecations)
+* [Reference](#reference)
+* [Copyright & License](#copyright--license)
+* [Reviews](#reviews)
+
+## Rationale
+
+Having two equivalent ways to do the same thing is a pointless redundancy in a language.
+Even worse, it engenders bikeshedding debates about which is "better". It's better
+to have one way of doing things, which reduces the complexity of the compiler, the specification,
+and efforts to teach the language.
+
+## Prior Work
+
+This DIP presumes the acceptance of [DIP: Named Arguments](https://github.com/dlang/DIPs/pull/168).
```

Comment:
Is Named Arguments really a prerequisite for this?  Why?

Reply:
Ok, I think I understand now.  This example...

```D
struct S { int a, b; }
S s = { 1, 2 }; // Deprecated: use S(1, 2) instead
```
.. should read ...
```D
struct S { int a, b; }
S s = { a:1, b:2 }; // Deprecated: use S(a:1, b:2) instead
```

Reply:
that's right